apps/aecore/src/aec_accounts_trees.erl

```diff
@@ -53,6 +59,16 @@ empty_with_backend() ->
 new_with_backend(Hash) ->
     aeu_mtrees:new_with_backend(Hash, aec_db_backends:accounts_backend()).
 
+-spec total_balances(tree()) -> integer().
+total_balances(Tree) ->
```

Comment:
This should use an iterator, and it is probably better to send in a limit to short circuit the traversal. This is an excellent attack vector otherwise.